src/config.js

```diff
@@ -177,9 +177,25 @@ export default class Config {
     await fs.mkdirp(this.globalFolder);
     await fs.mkdirp(this.cacheFolder);
     await fs.mkdirp(this.tempFolder);
-
     await fs.mkdirp(this.linkFolder);
-    this.linkedModules = await fs.readdir(this.linkFolder);
+
+    this.linkedModules = [];
+
+    const maybeLinkedModules = await fs.readdir(this.linkFolder);
+
+    for (const maybeLinked of maybeLinkedModules) {
+      const linkedPath = path.join(this.linkFolder, maybeLinked);
+      const isScopedName = maybeLinked.indexOf('@') === 0;
+      const isScopedContainer = isScopedName && !await fs.exists(path.join(linkedPath, 'package.json'));
```

Comment:
Hardcoding `package.json` isn't the best idea since internally we're really agnostic. I'd recommend leaving out this check since you can't have a package named `@foo` without having a slash in it anyway.